packages/core/src/components/forms/numericInput.tsx

```diff
@@ -594,4 +666,12 @@ export class NumericInput extends AbstractPureComponent<HTMLInputProps & INumeri
         const scaleFactor = Math.pow(10, this.state.stepMaxPrecision);
         return Math.round(value * scaleFactor) / scaleFactor;
     }
+
+    private updateDelta(
+        direction: IncrementDirection,
+        e: React.MouseEvent<HTMLInputElement> | React.KeyboardEvent<HTMLInputElement>,
```

Comment:
`React.SyntheticEvent` instead of union

Reply:
Tried that, but had to settle for the union because `shiftKey` and `altKey` aren't defined on [`SyntheticEvent`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L437).

Reply:
ok fair